app/models/member.rb

```diff
@@ -157,6 +157,11 @@ def party_name
     party_object.long_name
   end
 
+  # Long version of party name ukr_version
+  def display_party_name_ukr
```

Comment:
This seems to be a repetition of the helper method so I think we should remove it and just use that.